src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -40,10 +43,239 @@ public void LanguageVersion()
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void FromMetadata()
+        {
+            var source =
+@"interface I
+{
+    void F1(System.IntPtr x, nint y);
+    void F2(System.UIntPtr x, nuint y);
+}";
+            var comp = CreateCompilation(source, parseOptions: TestOptions.RegularPreview);
```

Comment:
>var method = model.GetDeclaredSymbol(methodDeclarations[0]); [](start = 12, length = 60)

I am not sure why SemanticModel is used to get to method symbols, they are easily accessible through symbol model, we have helpers like GetMember, etc.